### PR TITLE
Add landing page for widget

### PR DESCRIPTION
**Pull request by DougReeder**
for someone who is trying to get started with an app, and doesn't care about wider issues (yet)

Implements #18

### CODE REVIEW

**Comment by rosano** [comment]
@DougReeder thanks for doing this 🙏🏽 I'd like to also take a closer look but will have space only next week as I'm currently in podcast mode and also travelling. Looking forward to this.

**Review by rosano — APPROVED** [review]
I think the Getting Started page could have a more clear sequence of steps (not sure how the points relate linearly) or maybe a more 'welcoming' vibe, but this is definitely more friendly and informative than the servers page.

File: get.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+nav_order: 2
+---
+
+# Getting Started
+
+1. Sign up for a remoteStorage account
+    - If a friend or administrator has set up an account for you, get the details from them.
+    - [5apps](https://5apps.com/storage/beta) is a commercial
+     remoteStorage provider that currently offers free storage accounts.
+2. If you navigated here from an app, it's waiting for you in another tab!
```
[the remark this answers is not in the document]
Comment (DougReeder) [reply get.md:13]: 1. User navigates to RS webapp
2. User clicks "Connect your storage"
3. User clicks "Need help?"
4. User creates account on 5apps or elsewhere. Successful signup ends on a page with a message such as "You've signed up. Thanks, username" (Armadietto) or something analogous for 5apps.

Ideal result: user has link back to original app (analogous to OAuth flow)
Actual result: user must remember they were creating the account for an app and which browser tab it's in.

This is a high-friction experience, in a world of lower-friction experiences.

[the remark this answers is not in the document]
Comment (rosano) [reply get.md]: Ok, seems like it's referring to the moment at point 4. I've re-worded it to feel more like a concrete step the person can take.

There's definitely friction there but it comes from not having a centralized place to create accounts. If people will always sign up at a host of their choosing, the app cannot suggest or maintain a link back. One possibility could be to pass the referring url onto each of the server links from the `/get` page.

**Review by raucao — COMMENTED** [review]
Great! This already looks much more welcoming to me than being thrown on a "servers" page.

I think what's missing is one sentence before the "Getting Started" to explain that the app they came from allows them to use their own storage account and that this :point_down: is how to get started.

**Comment by rosano** [comment]
@raucao @DougReeder just added that in the last commit. I made PR #72 with even more explanations, I think it's helpful.

**Comment by DougReeder** [comment]
The emoji and rewording help!

I'd prefer to tighten it up, by replacing "Here are the steps..." with "Welcome!" and deleting "A more extensive list of options..." (The Servers page can still be accessed from the navigation).

But if you want to go with this wording, I'm okay with it.

**Comment by rosano** [comment]
@raucao seems like the changes from pull request #72 were squashed in the force push, is there a way to see the individual commits again?

**Comment by DougReeder** [comment]
I didn't want master to have a lot of little commits.  You'll have to pick them out by their hashes:

commit 971d325a17d89947a71d611d633014ee208105f2 (HEAD -> widget-landing)
Author: P. Douglas Reeder <reeder.29@gmail.com>
Date:   Wed Feb 2 20:17:33 2022 -0500

    tweaks description

commit ac1560e8bc9df4a54b6e059b2101c459602815af (origin/widget-landing)
Merge: 45c1aa4 df015e8
Author: Doug Reeder <reeder.29@gmail.com>
Date:   Wed Feb 2 19:58:08 2022 -0500

    Merge pull request #72 from remotestorage/widget-landing-verbose
    
    Widget landing verbose

commit df015e84f94fe7aff076e1b212acf78dd6e61eee
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 16:44:53 2022 -0500

    Fix typo

commit 5c28ef005e69d143c23c8b4daefc1a0e6285cfaa
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 16:40:16 2022 -0500

    Emojify

commit 41d037487754a4903b8003b618ca4a1b2e3a3903
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 16:38:30 2022 -0500

    Bold steps

commit d4c8bf1d9546f1424a24f370e3f305f17ed32bfc
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 16:38:11 2022 -0500

    Provide context relating to traditional web services

commit 45c1aa4dbd74eaa1e0c1632ca0b76a2f3c88b961
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 16:26:56 2022 -0500

    Add preface to steps with emoji

commit be6202295f59ca1c409f0ce2014081d9d72abbc8
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 15:33:55 2022 -0500

    Re-phrase as action step

commit d0ba1ffd8e5c48d7a739a0cb6705e74c25b4dedc
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 15:33:22 2022 -0500

    Link to servers page

commit fe01f54501295e578cac4fa15a4a36b94d104c9e
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 15:26:47 2022 -0500

    Lint

commit 796713e3644f12615445b826f08ff970a03cbf13
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 15:26:12 2022 -0500

    Fix typo

commit f7157cb383b0f5f6e2d5cf061eb39dda60a4cf94
Merge: 0ac9a66 b642831
Author: Doug Reeder <reeder.29@gmail.com>
Date:   Wed Feb 2 10:52:22 2022 -0500

    Merge pull request #70 from remotestorage/patch-1
    
    Revise getting started

commit b642831f6c3937c54734c3999f8ca3a9b2ef2208 (origin/patch-1, patch-1)
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 10:37:08 2022 -0500

    Fix typo

commit 76fba86ff85149d0b33aeccfe579a98febbdcf4a
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 10:34:52 2022 -0500

    Stylize RemoteStorage

commit 2a52ef27651cba9c8826f24c76f598c8a5fd605d
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 10:33:57 2022 -0500

    Move widget notice to /get comment

commit 7790651e4421974390b16a48b583532103349692
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 10:30:43 2022 -0500

    Move getting-started to get

commit 619495ecb1e002b77599153c836954247f13330e
Author: Rosano Coutinho <pub@xeari.com>
Date:   Wed Feb 2 10:29:56 2022 -0500

    Remove unreferenced ignore

commit 0ac9a667d310743d1200d14b4ce03851e78ddf3f
Author: P. Douglas Reeder <reeder.29@gmail.com>
Date:   Wed Jan 26 05:16:29 2022 -0500

    Adds landing page for widget

**Comment by rosano** [comment]
I guess this is a stylistic thing (small versus large commits) but I think it's nice to have the option for both legible on GitHub (without needing the command line) for GUI people like me. I always imagined pull requests as a sort of 'large commit' and smaller commits inside to denote the traceable steps involved in changes… But it's your pull request, so as you wish. IMO this is ready to merge so I'll leave it to @raucao to take final look.

**Review by raucao — APPROVED** [review]
LGTM :+1:

**Comment by rosano** [comment]
BTW I deleted a comment because they contained email addresses.

**Pull request merged**